vue3-apollo-quasar/src/components/IssuesPullRequestsCard/IssuesPullRequestsCard.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <q-card flat>
+    <q-card-section class="row no-wrap q-pa-sm">
+      <span :class="cardType">
+        <q-icon class="text-body1" :class="state">
+          <IssuesIcon v-if="cardType === 'issue' && state === 'open'" />
+          <ClosedIssuesIcon
+            v-else-if="cardType === 'issue' && state === 'closed'"
+          />
+
+          <PullRequestsIcon
+            v-if="cardType === 'pullrequest' && state === 'open'"
```

Comment:
> I would propose to check this: https://forum.quasar-framework.org/topic/6400/mapping-custom-svg-icons
> 
> It will help to avoid creating a component per icon. It seems like effective...

We could create a ticket for that purpose. Thanks.

Reply:
OKay, that can be fixed